src/workers/helper.py

```diff
@@ -1,11 +1,20 @@
-from datetime import datetime
+from datetime import datetime, timezone, timedelta
+import logging
+import asyncio
 import discord
+from discord.utils import snowflake_time
 from discord.ext.commands import Bot
 from settings import get_settings
 from util import create_interaction_embed_context
 from typing import Optional
 
+
+class UnableToDM(Exception):
+    pass
```

Comment:
i think this would go better in util.py and we can raise a `RunTime` error here

Reply:
If we aren't using this and using `RunTime` error instead then shouldn't I just remove this completely instead of moving it to `util.py`? or am I misunderstanding

Reply:
Oh like move this to util.py then you can raise the RunTime error inside the class so it would raise a UnableToDm error and a RunTimeError